target/linux/layerscape/image/Makefile

```diff
@@ -46,10 +59,30 @@ define Build/append-ls-dtb
 endef
 
 define Build/append-ls-rootfs-ext4
-	$(STAGING_DIR_HOST)/bin/make_ext4fs -l $(word 2,$(1)) -b 4096 -i 6000 -m 0 -J $(KDIR)/$(word 1,$(1))-$(word 2,$(1)).root.ext4 $(TARGET_DIR)
+	$(STAGING_DIR_HOST)/bin/make_ext4fs -l $(word 2,$(1)) -b 4096 -i 6000 -m 0 -J \
+		$(KDIR)/$(word 1,$(1))-$(word 2,$(1)).root.ext4 $(TARGET_DIR)
```

Comment:
There's no change here right?

Reply:
yes, there is no change, only split it into two lines.